unlocks.csv

```diff
@@ -452,17 +443,9 @@ Goal|OnCraftConstructor|0,0,Constructor|TitaniumIngot|Lubricant|Silicone|PowerCe
 Goal|OnConstructCyclops|0,0,CyclopsBridgeBlueprint|CyclopsEngineBlueprint|CyclopsHullBlueprint|Constructor|TitaniumIngot|PlasteelIngot|EnameledGlass|Glass|Lubricant|AdvancedWiringKit|WiringKit|ComputerChip|CopperWire
 Goal|OnConstructCyclops|1,0,CyclopsBridgeBlueprint|CyclopsEngineBlueprint|CyclopsHullBlueprint|Constructor|TitaniumIngot|PlasteelIngot|EnameledGlass|Glass|Lubricant|AdvancedWiringKit|WiringKit|ComputerChip|CopperWire
 Goal|OnConstructCyclops|2,0,CyclopsBridgeBlueprint|CyclopsEngineBlueprint|CyclopsHullBlueprint|Constructor|TitaniumIngot|PlasteelIngot|EnameledGlass|Glass|Lubricant|AdvancedWiringKit|WiringKit|ComputerChip|CopperWire
-Goal|OnConstructCyclops|3,1,CyclopsBridgeBlueprint|CyclopsEngineBlueprint|CyclopsHullBlueprint|Constructor|TitaniumIngot|PlasteelIngot|EnameledGlass|Glass|Lubricant|AdvancedWiringKit|WiringKit|ComputerChip|CopperWire
-Goal|OnConstructCyclops|4,1,CyclopsBridgeBlueprint|CyclopsEngineBlueprint|CyclopsHullBlueprint|Constructor|TitaniumIngot|PlasteelIngot|EnameledGlass|Glass|Lubricant|AdvancedWiringKit|WiringKit|ComputerChip|CopperWire
+Goal|OnConstructCyclops|3,0,CyclopsBridgeBlueprint|CyclopsEngineBlueprint|CyclopsHullBlueprint|Constructor|TitaniumIngot|PlasteelIngot|EnameledGlass|Glass|Lubricant|AdvancedWiringKit|WiringKit|ComputerChip|CopperWire
```

Comment:
Again, similar as for the PowerUpgradeModule, these are technically normal goals - the Efficiency module and Decoy tube normally unlock on construction. But there is already a Decoy tube databox. I think it is kinder to not require the Cyclops being constructed as standard checks.